src/chocolatey/infrastructure/powershell/PoshHostUserInterface.cs

```diff
@@ -129,7 +129,7 @@ public override void WriteProgress(long sourceId, ProgressRecord record)
             }
 
             // http://stackoverflow.com/a/888569/18475
-            Console.Write("\rProgress: {0}% - {1}".format_with(record.PercentComplete.to_string(), record.StatusDescription.PadRight(50, ' ')));
+            Console.Write("\rProgress: {0}% - {1}".format_with(record.PercentComplete.to_string(), record.StatusDescription).PadRight(Console.WindowWidth));

```

Comment:
I can repro it. Let me find the issue and why you want to use Console adapter. Bonus is you are probably using it and don't realize it (unless you went and clicked on go to implementation aleady).